webtop/__init__.py

```diff
@@ -222,13 +232,10 @@ def shutdown_signal_handler(_, __):
     tasks = []
 
     if args.duration is not None:
+        duration = durationpy.from_str(args.duration)
```

Comment:
Casting this here and not in the arg parser because I wanted an invalid format string to error out in the assert. Feels cleaner to me and allows us to add more useful failure info in the future.